lib/common/domain/check_if_app_terms_should_be_shown_usecase.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:zachranobed/services/auth_service.dart';
+import 'package:zachranobed/features/appConfiguration/data/get_last_app_terms_version_use_case.dart';
+
+class CheckIfAppTermsShouldBeShownUseCase {
+  final AuthService authService;
+  final GetLastAppTermsVersionUseCase _getLastAppTermsVersionUseCase;
+
+  CheckIfAppTermsShouldBeShownUseCase(
+      this.authService,
+      this._getLastAppTermsVersionUseCase
+  );
+
+  Future<bool> checkIfAppTermsShouldBeShown() async {
```

Comment:
Q: How use-cases look like on iOS? On Android we do call this method simply "invoke", or "execute", so that it is not "duplicating" on call site:

```dart
_checkIfAppTermsShouldBeShownUseCase.checkIfAppTermsShouldBeShown()
``` 

vs.

```dart
_checkIfAppTermsShouldBeShownUseCase.invoke()
```

Reply:
On iOS, we are used to name UC methods according to the action in more described way, however, I also prefer to use `invoke` since the UC name should do the job and we know, that UC should only have one specific non-private method

Reply:
Changed